### DISCUSSION TITLE
use Quectel as UE for a system setup following tutorial KPM xApp on FlexRIC

### DISCUSSION THREAD

**Discussion by initChat** [srsran/srsRAN_Project#429] [General Help] [answered]
I have successfully run this [tutorial on deployed KPM xApp on FlexRIC](https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html). 
The system has 
- open5GS as core (from docker of srsRAN_Project)
- FlexRIC as RIC
- srsRAN_Project use for simulating gNB
- srsRAN_4G uses for simulating srsUE.
![Screenshot from 2024-01-26 14-54-36](https://github.com/srsran/srsRAN_Project/assets/40858099/fbfd043c-4b60-4146-878c-cad68206a3b3)

I wonder that if I want to use Quectel RM510Q-GL as a UE. In this setup, Which files need to configuration? On gNB or on Open5GS.

Also, I only has bladeRF A4, which I successfully run 4G O-RAN by srsRAN_4G. But I cannot make it for 5G. So right now I do not have gNB for srsRAN_Project.

Thank you very much.

**Comment by pgawlowicz** [comment, marked as answer]
You need to run the gNB with an USRP. Basically, you need to follow this tutorial: https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html and enable the E2 agent in the gNB config file.

**Reply by pgawlowicz** [reply]
Currently, there is no support for bladeRF boards, only the USRP family.

**Reply by initChat** [reply]
Do I understand correctly that Quectel need SDR to connect to the network?

**Reply by pgawlowicz** [reply]
Yes, USRP SDR or a RU. Check here; https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/hw_integration.html

**Reply by initChat** [reply]
thank you very much. appreciate your help.
